### PR TITLE
Feature/TR-4575/Render maths expression in prompt using MathQuill

### PR DESCRIPTION
**Pull request by jsconan**
### Summary
Bring support to Math Entry PCI for rendering Math expressions in the prompt.

### Details
MathJax cannot be bundled with a PCI as it is not properly scoped. For this reason, we need to use another renderer for having Maths expressions in the prompt of the PCI.

This PCI is already including such a rendered. Indeed, the Maths expressions are managed by MathQuill, which supports LaTeX.

For rendering Maths in prompt, we can use it as well. However, need to make sure the expressions are expressed in LaTeX only. A 3rd-party library is needed to convert MathML. This is done thanks to [`mathml-to-latex`](https://github.com/qtikit/mathml-to-latex).

### How to test
- create/update an item and a Math Entry PCI
- add a Math expression in the prompt (better to add both LaTeX and MathML expressions for better testing)
- save, and edit again: the Math expression should be rendered properly
- create/update a test with the ite,
- publish the test and take the test: the Math expression should render fine

### Technical details
- Even if MathJax is not used by the feature, you still have to install it. Otherwise, the authoring won't propose the Math expression container.
- To add a Math expression to the prompt, you need to select the tool in the editor toolbar, then you can insert the Math expression in the input field from the right side panel.
image

### CODE REVIEW

File: migrations/Version202209141533261465_qtiItemPci.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        return 'Update the Math Entry PCI, adding support for rendering Math expressions in the prompt';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addReport(
+            $this->propagate(
+                new RegisterPciMathEntry()
+            )(
+                ['2.2.0']
```
Comment (marpesia) [inline migrations/Version202209141533261465_qtiItemPci.php:29]: Please, check the last PCI version before merging it. Several tickets are developing with this PCI at the moment.

**Review by marpesia — APPROVED** [review]
It works great

**Review Checklist**
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [ ] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] Commits messages are meaningful
- [ ] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

**Review by oatymart — APPROVED** [review]
Good job, I see it working correctly for valid LaTeX and MathML inputs in the PCI prompt. The dependency bundling and its instructions are very clear, and the enhanced test is nice too.

- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] E2E tests are running successfully (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Documentation is updated according to changes (if applicable)

**Comment by AntonNeviarouski** [comment]
QA verified

**Pull request merged**